src/components/IndividualArticle.jsx

```diff
@@ -0,0 +1,65 @@
+import { useEffect, useState } from "react";
+import { getArticleByID } from "../../api.js";
+import { useParams } from "react-router-dom";
+import { useContext } from "react";
+import { CurrentPageLabelContext } from "../contexts/CurrentPageLabel.jsx";
+
```

Comment:
It would be great to see a loading pattern here. We need some loading feedback for a better UX - let the user know that something is happening/data is being requested 💡